<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Network</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pbOpenRoutes">
     <property name="text">
      <string>Routes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="leSeedOD"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pbOpenSeedOD">
     <property name="text">
      <string>Seed OD</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="leRoutes"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="leLinks"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="leNodes"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbOpenLinks">
     <property name="text">
      <string>Links</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="leTurns"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pbOpenTurns">
     <property name="text">
      <string>Turns</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbOpenNodes">
     <property name="text">
      <string>Nodes</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="leZoneTargets"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pbOpenZoneTargets">
     <property name="text">
      <string>Zone Targets</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leNodes</tabstop>
  <tabstop>leLinks</tabstop>
  <tabstop>leTurns</tabstop>
  <tabstop>leRoutes</tabstop>
  <tabstop>leSeedOD</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
